stdlib/public/runtime/ImageInspection.h

```diff
@@ -30,6 +30,23 @@ namespace swift {
     void *symbolAddress;
   };
 
+#if defined(__ELF__) || defined(__ANDROID__)
+
+struct SectionInfo {
+  uintptr_t size;
+  const char *data;
+};
+
+// These are defined in swift_sections.S
+extern "C" const SectionInfo __swift2_protocol_conformances_block;
+extern "C" const SectionInfo __swift2_type_metadata_block;
+
+// Called by injected constructors when a dynamic library is loaded
+void addImageProtocolConformanceBlock(const SectionInfo *block);
+void addImageTypeMetadataRecordBlock(const SectionInfo *block);
+
+#endif // defined(__ELF__) || defined(__ANDROID__)
```

Comment:
I don't think this is worth changing. We're wasting space for pointers that can be trivially recovered.